clang/lib/Frontend/CompilerInstance.cpp

```diff
@@ -955,8 +955,10 @@ bool CompilerInstance::ExecuteAction(FrontendAction &Act) {
   // Adjust target options based on codegen options.
   getTarget().adjustTargetOptions(getCodeGenOpts(), getTargetOpts());
 
-  if (auto *Aux = getAuxTarget())
+  if (auto *Aux = getAuxTarget()) {
```

Comment:
I see the 'createPreprocessor' has a similar target.adjust, does this need to be changed there too?

Reply:
To be honest, I don't know. I just tried to comment it out and it seems that check-clang passes with such change. Not sure that it has any effect there.
My intention was to find a place where aux target is connected with the current target and adjust aux target there. I found only this one.

Reply:
Hmm... interesting.  While I think it is likely valuable to be in the prepreprocessor function as well, I don't think much happens for the preprocessor for target-specific actions.

That said, if this fixes mlong-double-#, it is likely sufficient.